.github/labeler.yml

```diff
@@ -0,0 +1,12 @@
+# Rules defined here: https://github.com/actions/labeler
+type/chore:
+  - ./*.md
+  - ./**/*.yml
+  - ./acceptance/**/*
+  - ./.github/**/*
+  - ./go.mod
+  - ./go.sum
+
+type/enhancement:
+  - any: ['./*.go', './**/*.go', '!./acceptance/**/*', '!./test*/**/*']
```

Comment:
This (I believe) should match any adjustment to a `.go` file, as long as it isn't just a change to the `acceptance` dir, or any top-level dir starting with `test` (like `testdata`, `testhelpers`, or `testmocks`). 

However, if any of the files in the PR are a change in the `.github` directory, it won't be labeled `type/enhancement`